src/wazuh_modules/syscollector/network/networkWindowsWrapper.h

```diff
@@ -172,24 +169,27 @@ class NetworkWindowsInterface final : public INetworkInterfaceWrapper
 
     std::string gateway() const override
     {
-        std::string retVal { "unknown" };
+        std::string retVal;
+        constexpr auto GATEWAY_SEPARATOR { "," };
         if (Utils::isVistaOrLater())
         {
-            const auto gatewayAddress { m_interfaceAddress->FirstGatewayAddress };
-            if (gatewayAddress)
+            auto gatewayAddress { m_interfaceAddress->FirstGatewayAddress };
+            while (gatewayAddress)
             {
                 const auto gatewayFamily { gatewayAddress->Address.lpSockaddr->sa_family };
                 const auto sockAddress   { gatewayAddress->Address.lpSockaddr };
                 if (AF_INET == gatewayFamily)
                 {
-                    retVal = Utils::NetworkWindowsHelper::IAddressToString(gatewayFamily, 
+                    retVal += Utils::NetworkWindowsHelper::IAddressToString(gatewayFamily,
                                                                            (reinterpret_cast<sockaddr_in*>(sockAddress))->sin_addr);
                 }
                 else if (AF_INET6 == gatewayFamily)
                 {
-                    retVal = Utils::NetworkWindowsHelper::IAddressToString(gatewayFamily, 
+                    retVal += Utils::NetworkWindowsHelper::IAddressToString(gatewayFamily,
                                                                            (reinterpret_cast<sockaddr_in6*>(sockAddress))->sin6_addr);
                 }
+                retVal += GATEWAY_SEPARATOR;
```

Comment:
move it inside the if because it will add a separator always.

Reply:
Will do.